README.md

```diff
@@ -32,7 +32,7 @@ When running the Strimzi canary tool, it is possible to configure different aspe
 * `TLS_CLIENT_KEY`: TLS client private key, in PEM format, to use for enabling TLS client authentication against the Kafka cluster. Default empty.
 * `TLS_INSECURE_SKIP_VERIFY`:  if the underneath Sarama client has to verify the server's certificate chain and host name. Default `false`.
 * `SASL_MECHANISM`: mechanism to use for SASL authentication against the Kafka cluster. Default empty.
-* `SASL_USER`: username for SASL authentication against the Kafka cluster when PLAIN or SCRAM-SHA are used. Default empty.
+* `SASL_USER`: username for SASL authentication against the Kafka cluster when PLAIN or SCRAM-SHA are used. Supported is `PLAIN`. Default empty.
```

Comment:
Seems a bit weird. Should you remove the other SCRAM-SHA reference? Since when Kafka cluster uses SCRAM-SHA you will not connect with PLAIN.

Reply:
Sorry wrong place!

Reply:
@scholzj fixed.